### PR TITLE
Allow mono debugging and jit flags

### PR DESCRIPTION
**Pull request by lostmsu**
Implements https://github.com/pythonnet/clr-loader/issues/13

### CODE REVIEW

File: clr_loader/__init__.py [hunk 1]
```diff
@@ -20,6 +22,8 @@ def get_mono(
 
     impl = Mono(
         domain=domain,
+        debug=debug,
+        jit_options=jit_options,
```
Comment (filmor) [inline clr_loader/__init__.py:26]: Seeing the parameters are also positional, please append the new ones at the end of the existing list.

File: clr_loader/mono.py [hunk 2]
```diff
@@ -113,6 +119,13 @@ def initialize(
 
         config_encoded = config_file.encode("utf8")
 
+        if jit_options:
+            options = list(map(lambda x: ffi.new("char[]", x.encode("utf8")), jit_options))
```
Comment (filmor) [inline clr_loader/mono.py:123]: Just use a list comprehension.

File: tests/test_common.py [hunk 3]
```diff
@@ -31,6 +31,15 @@ def test_mono(example_netstandard):
 
     run_tests(asm)
 
+def test_mono_debug(example_netstandard):
+    from clr_loader import get_mono
+
+    mono = get_mono(debug=True,
+                    jit_options=['--debugger-agent=address=0.0.0.0:5831,transport=dt_socket,server=y'])
```
Comment (filmor) [inline tests/test_common.py:38]: Please run the black formatter over the changes.

**Comment by lostmsu** [comment]
@filmor updated

File: clr_loader/__init__.py (hunk 1 above)
Comment (lostmsu) [reply clr_loader/__init__.py:26]: Updated in `get_mono`, kept as is for `Mono.__init__` as in the later parameters are keyword only

**Review by filmor — APPROVED** [review]

**Pull request merged**